### PR TITLE
fix: correct location for several plugins

### PR DESCRIPTION
**Pull request by leos**
## Summary

The old URL redirects to the new one so this saves a redirect hop

## Checklist

- [ ] CI tests are green. If you are using GitHub, you might want to use the `plugin_test` action from [asdf-actions](https://github.com/asdf-vm/actions)
- [ ] `asdf-plugins` CI sanity checks are green on your PullRequest. Test locally with:

```bash
./test_plugin.sh --file plugins/<PLUGIN_FILE>
```

### CODE REVIEW

**Comment by jthegedus** [comment]
Can you update the refs/links in the README?

**Comment by leos** [comment]
@jthegedus done - I also cleaned up a few other packages that were showing `build:unknown` because they were pointing at old github repos that were moved.

File: README.md [hunk 1]
```diff
@@ -329,8 +329,8 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
...
 | Levant                        | [asdf-community/asdf-hashicorp](https://github.com/asdf-community/asdf-hashicorp)                               | [![Build](https://github.com/asdf-community/asdf-hashicorp/actions/workflows/workflow.yml/badge.svg)](https://github.com/asdf-community/asdf-hashicorp/actions/workflows/workflow.yml)                                            |
-| LFE                           | [vic/asdf-lfe](https://github.com/vic/asdf-lfe)                                                                 | [![Build Status](https://travis-ci.org/vic/asdf-lfe.svg?branch=master)](https://travis-ci.org/vic/asdf-lfe)                                                                                                                       |
-| Link (system tools)           | [vic/asdf-link](https://github.com/vic/asdf-link)                                                               | [![Build Status](https://travis-ci.org/vic/asdf-link.svg?branch=master)](https://travis-ci.org/vic/asdf-link)                                                                                                                     |
```
Comment (jthegedus) [inline README.md:333]: This one doesn't appear to be in the `plugins/` dir. Can you confirm?

**Comment by jthegedus** [comment]
> @jthegedus done - I also cleaned up a few other packages that were showing `build:unknown` because they were pointing at old github repos that were moved.

Thanks, really appreciate it. Just one more question about one plugin

File: README.md (hunk 1 above)
Comment (leos) [reply README.md:333]: I don't see it there, but I have no idea what this is and/or whether it should be. This is the first time I'm using `asdf` and just figured I'd contribute these small redirect fixes.

Or are you asking someone else to confirm it?

Comment (jthegedus) [reply README.md:333]: The readme is not what is used by `asdf`, but rather just information for users to look at to ctrl+f search for tools etc. `asdf` uses the files in `plugins/*` dir to download the plugins. The "Link" plugin you've updated here doesn't have a matching file in `plugins/*` so won't actually be installable via this repo

Comment (leos) [reply README.md:333]: That makes sense. What would you like me to do in this PR then?

I could:
* not update that entry in the README
* remove it from the README
* add it a file in the plugins directory (not sure what process you follow, but that's probably best in a separate PR?)
* something else?

Comment (jthegedus) [reply README.md:333]: I mean, if it exists, add it.

**Review by jthegedus — APPROVED** [review]

**Comment by jthegedus** [comment]
Merging so I can unblock merging the other PRs

**Pull request merged**